src/components/_checkbox-new.scss

```
@@ -0,0 +1,202 @@
+// Checkbox (new)
+//
+// New checkbox component, more flexible (no dependent of a DOM label)
+//
+// Browser support: IE11+, Safari6+, Firefox30+, Chrome30+
+//
+// Markup:
+// <div class="sb-checkbox-new sb-margin-xs">
+//   <input type="checkbox" id="checkbox-1" class="sb-checkbox-new__input">
+//   <div class="sb-checkbox-new__box">
```

```suggestion
//   <div class="sb-checkbox-new__checkmark-container">
```
or
```suggestion
//   <div class="sb-checkbox-new__checkmark-block">
```
maybe something more meaningful not `__box`